docs/src/data/ignoredLinks.ts

```diff
@@ -10,5 +10,12 @@ export const IGNORED_LINKS = [
   'https://docs.aws.amazon.com/pinpoint/latest/userguide/channels-inapp.html', // 403
   'https://docs.aws.amazon.com/amplify/latest/userguide/getting-started.html', // 403
   'https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-attributes.html', // 403
+  'https://docs.aws.amazon.com/general/latest/gr/rekognition.html', // 403,
+  'https://docs.aws.amazon.com/rekognition/latest/dg/face-liveness-programming-api.html', // 403
+  'https://docs.aws.amazon.com/rekognition/latest/dg/recommendations-liveness.html', // 403
   'https://github.com/aws-amplify/amplify-ui/issues/new/choose', // 302. Only at test. Can't reproduce the status code. Can't find the redirected link.
+  'https://github.com/aws-amplify/amplify-ui-android/issues/new/choose', // 302
+  'https://github.com/aws-amplify/amplify-ui-swift/issues/new/choose', // 302
```

Comment:
why don't we use https://github.com/aws-amplify/amplify-ui-swift-liveness/issues/new ?

Reply:
Instead of ignoring those 2 links
'https://github.com/aws-amplify/amplify-ui-swift/issues/new/choose'
'https://github.com/aws-amplify/amplify-ui-swift-liveness/issues/new/choose'

we should update them
"https://github.com/aws-amplify/amplify-ui-swift-liveness/issues/new"
"https://github.com/aws-amplify/amplify-ui-swift-liveness/issues/new"


https://github.com/aws-amplify/amplify-ui-android/issues/new/choose can be added here since it's not really a 302

Reply:
Can we keep it that way if they are planning to add templates in the future?